admin_manual/release_notes.rst

```diff
@@ -5,9 +5,22 @@ Release Notes
 Changes in 10.0
 ---------------
 
-* Supported PHP versions are 5.6+ and 7.0+. 
-* The upgrade migration test has been removed; see :ref:`migration_test_label`.
-* Requires to use at least desktop client version 2.0 by default.
+* PHP 7.1 support added (supported PHP versions are 5.6 and 7.0+)
+* The upgrade migration test has been removed; see :ref:`migration_test_label`. (Option "--skip-migration-tests" removed from update command)
+* Requires to use the latest desktop client version 2.3
+* Settings design rework: Admin, personal pages and app management are now merged together into a single "Settings" entry
+* Third party apps are not disabled anymore when upgrading
+* DB Migrations for seamless/improved update process added (enables skipping major versions when upgrading after 10.0)
+* Ability to disable users in the user management panel added (users are prevented from logging in but the account stays as it is and can be enabled again) 
+* User account table has been reworked. CRON job for syncing with e.g. LDAP needs to be configured (see https://doc.owncloud.com/server/10.0/admin_manual/configuration_server/occ_command.html#syncing-user-accounts)
+* Ability to split app logs (e.g. auditing) and owncloud.log (see https://doc.owncloud.org/server/10.0/admin_manual/configuration_server/config_sample_php_parameters.html#logging)
```

Comment:
Note: auditing (admin_audit) is only avaiable on enterprise release, not sure if it should be mentioned here.

What can be said here is that it makes it possible redirect log entries from specific apps to separate log files. (and since admin_audit is an app, it can work with that too.